proxy/src/http/project.rs

```diff
@@ -676,6 +717,38 @@ mod test {
         Ok(())
     }
 
+    #[tokio::test]
+    async fn list_for_user() -> Result<(), error::Error> {
+        let tmp_dir = tempfile::tempdir()?;
+        let ctx = http::Context::tmp(&tmp_dir).await?;
+        let api = super::filters(ctx.clone());
+
+        let ctx = ctx.read().await;
+        let key = ctx.keystore.get_librad_key()?;
+        let owner = ctx.peer_api.init_owner(&key, "cloudhead")?;
+
+        coco::control::setup_fixtures(&ctx.peer_api, &key, &owner)?;
+        #[allow(clippy::indexing_slicing)]
+        let project = &project::list_projects(&ctx.peer_api)?[0];
+        let librad_project = ctx.peer_api.get_project(&project.id, None)?;
+
+        let fintohaps: identity::Identity =
+            coco::control::track_fake_peer(&ctx.peer_api, &key, &librad_project, "fintohaps")
+                .into();
+
+        let res = request()
+            .method("GET")
+            .path(&format!("/?user={}", fintohaps.urn))
```

Comment:
Are there no characters which need encoding in our URNs?

Reply:
Nope. We do this everywhere already as well.